tests/publiccloud/check_registercloudguest.pm

```diff
@@ -126,14 +126,14 @@ sub check_instance_registered {
         record_info('zypper lr', $instance->ssh_script_output(cmd => 'zypper -t lr ||:'));
         die('The list of zypper repositories is empty.');
     }
-    if ($instance->ssh_script_output(cmd => 'sudo ls /etc/zypp/credentials.d/ | wc -l') == 0) {
+    if ($instance->ssh_script_output(cmd => 'sudo ls -A /etc/zypp/credentials.d/') != 0) {
         die('Directory /etc/zypp/credentials.d/ is empty.');
     }
 }
 
 sub check_instance_unregistered {
     my ($instance, $error) = @_;
-    if ($instance->ssh_script_output(cmd => 'sudo ls /etc/zypp/credentials.d/ | wc -l') != 0) {
+    if ($instance->ssh_script_output(cmd => 'sudo ls -A /etc/zypp/credentials.d/') == 0) {
```

Comment:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20260/files#r1773481308

Reply:
Resolving this. We can discuss in the other comment.